keyboards/rhymestone/keymaps/default/keymap.c

```diff
@@ -0,0 +1,225 @@
+  //,---------------------------------------------------------------------------------------------------.
+         KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,  KC_MINS,   KC_EQL,  KC_LBRC,  KC_RBRC,  KC_BSLS,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+         KC_F6,    KC_F7,    KC_F8,    KC_F9,   KC_F10,  XXXXXXX,  XXXXXXX,  XXXXXXX,  KC_SCLN,  KC_QUOT,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       KC_11SF,   KC_F12,   KC_ESC,   KC_TAB,    KANJI,   KC_DEL,  XXXXXXX,  XXXXXXX,    KC_RO,  KC_GRSF,
+  //`---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'
+       _______,  _______,  _______,  _______,   KC_DEL,  _______,  _______,  _______,  _______,  _______
+  //,---------------------------------------------------------------------------------------------------.
+  ),
+
+  [_RAISE] = LAYOUT( \
+  //,---------------------------------------------------------------------------------------------------.
+          KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       KC_LSFT,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  KC_LEFT,  KC_DOWN,    KC_UP,  KC_RGHT,  KC_RSFT,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,  KC_MINS,    KC_RO,  KC_COMM,   KC_DOT,  KC_SLSF,
+  //`---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'
+       _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______
+  //,---------------------------------------------------------------------------------------------------.
+  ),
+
+  [_ADJUST] = LAYOUT( \
+  //,---------------------------------------------------------------------------------------------------.
+         RESET,   RGBRST,  AG_NORM,  AG_SWAP,  XXXXXXX,  XXXXXXX,  XXXXXXX,  XXXXXXX,   KC_INS,  KC_PSCR,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       RGB_TOG,  RGB_HUI,  RGB_SAI,  RGB_VAI,  XXXXXXX,  KC_MS_L,  KC_MS_D,  KC_MS_U,  KC_MS_R,  KC_NLCK,
+  //|---------+---------+---------+---------+---------+---------+---------+---------+---------+---------|
+       RGB_MOD,  RGB_HUD,  RGB_SAD,  RGB_VAD,  XXXXXXX,  KC_BTN1,  KC_BTN2,  XXXXXXX,  XXXXXXX,  XXXXXXX,
+  //`---------+---------+---------+---------+---------+---------+---------+---------+---------+---------'
+       _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______
+  //,---------------------------------------------------------------------------------------------------.
+  )
+};
+
+#define L_BASE _BASE
```

Comment:
Honestly, I'd like to see the entire OLED block moved to the keyboard's c file, with the `oled_task_user` function defined weakly.  

This way, it will work in the configurator, but can be customized locally too.